### PR TITLE
chore: updated brightway, fixed some versions for jupyter, fixed the random issue with ground-beef-organic

### PR DESCRIPTION
**Pull request by ccomb**
- updated brightway
- fixed versions for jupyter
- fixed the organic beef random bug
- installed elm in docker since it's used in data generation
- removed the unuseful prettier embedded in compute_aggregated

### CODE REVIEW

File: data/common/import_.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def add_average_activity(activity_data, dbname):
 def replace_activities(activity_variant, activity_data, dbname):
     """Replace all activities in activity_data["replace"] with variants of these activities"""
     for old, new in activity_data["replace"].items():
-        if isinstance(list, new):
+        if isinstance(new, list):
```
Comment (n1k0) [inline data/common/import_.py:95]: ouch 💀

File: public/data/food/processes.json [hunk 2]
```diff
@@ -8866,8 +8866,8 @@
       "swe": 0,
       "tre": 0,
       "wtu": 0,
-      "ecs": 163.7601782548028,
-      "pef": 299.5375550176106
+      "ecs": 1162.0606885562001,
+      "pef": 553.4152498406837
```
Comment (n1k0) [inline public/data/food/processes.json:8870]: That's quite a large score update /cc @paulboosz

**Review by n1k0 — COMMENTED** [review]
I'm pre-approving the code changes but I'll leave @paulboosz validate the resulting score updates.

**Comment by paulboosz** [comment]
I checked the differences between simapro-evea and simapro-christophe implementation of "remove azadirachtine"
the impacts are exactly the same 

[analysis](https://docs.google.com/spreadsheets/d/1lAZQ7OLwHRqUGpttnQrbXerxorG8fgxTDc7qzw6qhH8/edit?usp=sharing)

**Comment by vjousse** [comment]
How am I supposed to test this upgrade?

**Comment by ccomb** [comment]
> How am I supposed to test this upgrade?

One way is to run the full import/export with a `make` in the `data/` folder, provided you have all the source databases locally, and compare the resulting impacts. But this is long and boring.

**Comment by paulboosz** [comment]
Organic bug is fixed congrats Chris 🥳🥳

**Review by paulboosz — APPROVED** [review]

**Pull request merged**